avidemux_core/ADM_coreUtils/src/ADM_paramList.cpp

```diff
@@ -197,8 +198,9 @@ bool ADM_compressWriteToString(COMPRES_PARAMS *params,  char **str)
         case COMPRESS_SAME:  sprintf(tmp,"SAME");break;
         case COMPRESS_2PASS_BITRATE: sprintf(tmp,"2PASSBITRATE=%" PRIu32,params->avg_bitrate);break;
         case COMPRESS_AQ:    sprintf(tmp,"AQ=%" PRIu32,params->qz);break;
+        case COMPRESS_LOSSLESS: sprintf(tmp,"LOSSLESS=1");break;
```

Comment:
For me, it feels somewhat awkward to write `"general.params=LOSSLESS=1"` when `"general.params=LOSSLESS=0"` means exactly the same. Could you please explain the rationale?

With `"general.params=LOSSLESS"` we can exit` compressReadFromString()` early like in case of `"general.params=SAME"` (just not really used anywhere) and it is free from ambiguity, IMHO.